pkgs/development/python-modules/slither-analyzer/default.nix

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
We prefer to format this like
```
{ lib
, buildPythonPackage
, fetchPypi
, makeWrapper
, prettytable
, pythonOlder
, solc
}:
```

or even all on one line like
```
{ lib, buildPythonPackage, fetchPypi, makeWrapper, prettytable, pythonOlder, solc }:
```

Reply:
My problem with the first approach is that it should then be used everywhere, not just for the function's argument set. So I'll go with the second.